traffic_portal/package.json

```diff
@@ -50,12 +41,12 @@
     "angular-route": "1.8.0",
     "angular-sanitize": "1.8.0",
     "angular-ui-bootstrap": "0.14.3",
-    "bootstrap-sass": "3.3.6",
-    "es5-shim": "4.5.6",
+    "bootstrap-sass": "3.4.1",
+    "es5-shim": "4.6.4",
```

Comment:
Do we really need a shim for ES5? The only browser we truly support is Chrome, which added full support for ES5 70 major versions ago - which is a full decade behind us now, which in fact predates the very existence of this project. All other major browsers have full compliance as well ([source](https://caniuse.com/?search=es5)) with the exception of Opera Mini, which has an issue with `parseInt`.